compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -823,6 +823,12 @@ OMR::SymbolReferenceTable::findOrCreateDivCheckSymbolRef(TR::ResolvedMethodSymbo
    }
 
 TR::SymbolReference *
+OMR::SymbolReferenceTable::findOrCreateOverflowCheckSymbolRef(TR::ResolvedMethodSymbol *)
+   {
+   return findOrCreateRuntimeHelper(TR_overflowCheck, false, true, true);
```

Comment:
It's for use only aliasing in OMR::SymbolReference::getUseonlyAliasesBV. we want to make sure defs are not moved across the overflowCHK in case an exception throws and the catch block might get stale values.